### ISSUE TITLE
sortedKeys is only available in iOS 11.0 or newer

### ISSUE THREAD

**Issue by rosolowski** [nicklockwood/SwiftFormat#924]
After installation version `0.48.0` by swift package manager I'm getting this error:

```
/SwiftFormat/Sources/JSONReporter.swift:51:46: 'sortedKeys' is only available in iOS 11.0 or newer
```

This line:
```
encoder.outputFormatting.insert(.sortedKeys)
```

**Comment by nicklockwood** [comment]
Thanks for reporting, I'll get that fixed.

**Comment by nicklockwood** [comment]
@rosolowski this should be fixed in 0.48.1 - please verify

**Comment by rosolowski** [comment]
@nicklockwood Yeah, it looks like everything's working
